doc/source/whatsnew/v2.1.0.rst

```diff
@@ -339,7 +340,6 @@ Period
 - Bug in :meth:`arrays.PeriodArray.map` and :meth:`PeriodIndex.map`, where the supplied callable operated array-wise instead of element-wise (:issue:`51977`)
 - Bug in :func:`read_csv` not processing empty strings as a null value, with ``engine="pyarrow"`` (:issue:`52087`)
 - Bug in :func:`read_csv` returning ``object`` dtype columns instead of ``float64`` dtype columns with ``engine="pyarrow"`` for columns that are all null with ``engine="pyarrow"`` (:issue:`52087`)
-- Bug in incorrectly allowing construction of :class:`Period` or :class:`PeriodDtype` with :class:`CustomBusinessDay` freq; use :class:`BusinessDay` instead (:issue:`52534`)
```

Comment:
something went wrong when merging here, I presume you didn't mean to remove this line?

Reply:
> Shouldn't my PR only contain my contribution?

yes, exactly - whereas currently, it's showing that you also deleted another line

please check https://github.com/pandas-dev/pandas/pull/52214/files to verify that the PR only contains your changes

Reply:
@MarcoGorelli Looking at that URL, it looks correct to me. 

The PR only contains my changes. However, when I run into merge conflicts after other people's PRs are approved, do I need to add their changes when I examine the merge conflict diff, or remove them? In this case I removed them, which is why the line below is shaded in red. Previously I added them and that seemed to cause problems as well. hmmmm

- Bug in incorrectly allowing construction of :class:`Period` or :class:`PeriodDtype` with :class:`CustomBusinessDay` freq; use :class:`BusinessDay` instead (:issue:`52534`)

Reply:
depends on the merge conflict - if it's a whatsnew note, you probably need to select "keep both changes"

Reply:
Understood! Thank you for the clarity! I'll add the above entry back and keep this in mind moving forward!